### PR TITLE
Add script to check for orphaned example files

### PR DESCRIPTION
**Pull request by settermjd**
This PR adds a small shell script that checks if any of the example files (scripts) is used anywhere throughout the documentation.

### CODE REVIEW

File: bin/find-orphaned-files.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+echo "Checking for orphaned example files."
+echo
+
+for manual in "${MANUALS[@]}"; do
+    [[ ! -e "./modules/${manual}_manual/" ]] && echo "Cannot find ${manual} manual." && continue
+
+    echo "Checking the ${manual} manual"
+    find "./modules/${manual}_manual/examples/" \
+        \( -path "**/.DS_Store" -o -path "**/vendor" -o -path "**/.idea" -o -path "**/.gitkeep" \) -prune -o \
```
Comment (tboerger) [inline bin/find-orphaned-files.sh]: `.idea` should never be part of some sub folder, so IMHO you could drop that.

Comment (mmattel) [reply bin/find-orphaned-files.sh]: Generally correct, in this find statement, it is part of excluded directories to be searched in.
Whyever a directory named .idea is present, exclude it in the search.
Therefore it is ok to me.

Comment (tboerger) [reply bin/find-orphaned-files.sh]: .idea folders are project folders by intellij, which would never be in a sub directory.

Comment (mmattel) [reply bin/find-orphaned-files.sh]: fixed (removed)

**Comment by mmattel** [comment]
@settermjd 
I have removed `.idea` according to @tboerger comment above.
Additionally I added some text to improve usability.

**Pull request merged**